CONTRIBUTING.rst

```diff
@@ -0,0 +1,67 @@
+Releasing
+=========
+
+Summary
+-------
+
+- The sunspec-demo repository uses the `feature branch workflow <https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow>`_ process to control development.
```

Comment:
Let's pick consistent capitalization.  I think the three options would be:

- SunSpec-Demo
- SunSpec-demo
- sunspec-demo

Though, the distribution and import package names are `epcsunspecdemo`.  But let's not get hung up on that.

Reply:
I chose "SunSpec-Demo" for all situations where the application is named. And "sunspec-demo" is used when referring to the directory path.